### PR TITLE
Add the 2022q3 report for portmgr

### PR DESCRIPTION
**Pull request by rene0**

### CODE REVIEW

File: 2022q3/portmgr.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Contact: René Ladan <portmgr-secretary@FreeBSD.org> +
+Contact: FreeBSD Ports Management Team <portmgr@FreeBSD.org>
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages, and personnel matters.
+Below is what happened in the last quarter.
+
+Currently there are just over 30,500 ports in the Ports Tree. There are currently just under 2,800 open ports PRs of which 750 are unassigned.
+The last quarter saw 9,137 commits by 151 committers on the main branch and 589 commits by 61 committers on the 2022Q3 branch.
+Compared to two quarters ago, this means a slight increase in the number of ports, but also a slight increase in the number (unassigned) ports PRs and a slight decrease in the number of commits made.
```
Comment (pauamma) [inline 2022q3/portmgr.adoc:18]: s/in the number (unassigned)/in the number of (unassigned)/

File: 2022q3/portmgr.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+In the last quarter, we welcomed Felix Palmen (zirias@) as a new ports committer, welcomed back Akonori MUSHA (knu@), and said goodbye to Olli Hauer (ohauer@).
+We also welcomed Luca Pizzamiglio (pizzamig@) as an official member of portmgr.
+
+Some large changes in the Ports Tree were made during the last quarter:
+* "Created by" lines have been removed from the top of each Makefile, as a lot of those were outdated.
+* WWW: has been moved from each pkg-descr into each Makefile as a variable, below write-up is from Stefan Eßer (se@) who did the work:
```
Comment (pauamma) [inline 2022q3/portmgr.adoc:25]: s/variable, below write-up/variable; the below write-up/

File: 2022q3/portmgr.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+By moving the definition of these URLs into the Makefiles, maintainers are more likely to update the URL along with other port changes, and tools have easier access to them. 
+The URLs are now assigned to the WWW macro in the Makefile and can be queried with "make -V WWW" in the port directory.
+Tools that process the description contained in the package files still work because the "WWW:" lines at the end are generated from the WWW values in the Makefiles.
+
+During EuroBSDCon, portmgr had a discussion about improving the situation for kernel module packages.
+Various possibilities have been discussed.
+
+The following happened under the hood:
+* one new USES, "vala", was added.
+* Default version of Go got bumped to 1.19
```
Comment (pauamma) [inline 2022q3/portmgr.adoc:39]: s/Default/The default/

**Review by pauamma — COMMENTED** [review]
Very minor. Unless you disagree, I'll fix them as I merge, within the next few days.

**Comment by rene0** [comment]
The fixes look good to me, thanks for catching them.

**Pull request merged**